src/FormatNumber/Parser.elm

```diff
@@ -211,6 +214,67 @@ getDecimals locale digits =
             addZerosToFit min digits
 
 
+
+{- Joins parts into `Maybe Float`. It expects parts to be strings with only digits.
+
+   joinParts ("100", "235")
+   --> Just 100.235
+
+   joinParts ("", "534")
+   --> Just 0.534
+
+   joinParts ("243", "")
+   --> Just 243
+
+   joinParts ("", "")
+   --> Nothing
+
+   joinParts ("100,00", "243")
+   --> Nothing
+
+   joinParts ("10000", "24,3")
+   --> Nothing
+
+-}
+
+
+joinParts : ( String, String ) -> Maybe Float
```

Comment:
```suggestion
-}
joinParts : ( String, String ) -> Maybe Float
```

Reply:
Additional new lines were from Elm format. I ran it multiple time to make sure. I don't know if it is some weird bug there.